graphspace_python/api/endpoint/layouts.py

```diff
@@ -188,7 +191,9 @@ def update_graph_layout(self, graph_id, layout_id, layout_name=None, positions_j
 			data.update({'style_json': style_json})
 
 		layout_by_id_path = LAYOUTS_PATH.format(graph_id) + str(layout_id)
-		return self.client._make_request("PUT", layout_by_id_path, data=data).json()
+		return LayoutResponse(
+			self.client._make_request("PUT", layout_by_id_path, data=data).json()
```

Comment:
Here the layout object is being returned after parsing the json data from the response...anyways we have to get the json data from the response first to parse it